content/en/docs/installation/kubernetes.md

```diff
@@ -154,7 +137,7 @@ To install the cert-manager Helm chart:
 $ helm install \
   cert-manager jetstack/cert-manager \
   --namespace cert-manager \
-  --version v1.1.0 \
+  --version v1.2.0 \
   # --set installCRDs=true
```

Comment:
Why do we need to comment this out? I imagine some people might fail to read the above sentence (`Uncomment the relevant line in the next steps to enable this.`) and not uncomment it.